packages/app-store/routing-forms/getEventTypeRedirectUrl.ts

```diff
@@ -86,3 +90,9 @@ export function getAbsoluteEventTypeRedirectUrl({
 
   return `${origin}/${eventTypeRedirectUrl}?${allURLSearchParams}`;
 }
+
+export function getAbsoluteEventTypeRedirectUrlWithEmbedSupport(
```

Comment:
Ensures that `isEmbed` is passed and accordingly allows updating the event URL which needs /embed in the end for embed.